core/templates/components/summary-tile/learner-story-summary-tile.component.html

```diff
@@ -0,0 +1,110 @@
+<mat-card class="oppia-story-dashboard-card">
+  <a [href]="getStoryLink()"
+     target="{{ openInNewWindow ? '_blank' : '_self' }}"
+     class="oppia-story-link">
+    <div class="oppia-title-section" [ngStyle]="{'background-color': thumbnailBgColor}">
+      <img class="oppia-thumbnail-image" [src]="thumbnailUrl">
+      <div class="oppia-story-completion">
+        <img [src]="getStaticImageUrl('/learner_dashboard/star.svg')" alt="star image" class="oppia-svg-star-image" *ngIf="storyCompleted">
+        <p class="oppia-percent-story-completed" [ngClass]="{'oppia-story-completed-percent': storyCompleted}">
+          {{storyProgress}}% Complete
+        </p>
+      </div>
+    </div>
+    <div class="oppia-summary-section">
+      <span class="oppia-story-title">
+        {{storyTitle}}
+      </span>
+      <span class="oppia-topic-title">
+        {{topicName}}
+      </span>
+    </div>
+  </a>
+</mat-card>
+<style>
+  .oppia-story-title {
```

Comment:
For all the classes here, can you prepend the parent class name? (eg: `.oppia-story-dashboard-card .oppia-story-title`), so that the CSS only affects this component.

Reply:
Done.